backend/frontend/src/screens/HomeScreen.js

```diff
@@ -82,6 +82,14 @@ function HomeScreen(history) {
     }
   }, [t]);
 
+  useEffect(() => {
+    const userName = localStorage.getItem("userName");
+    if (userName) {
+      toast.success(`Welcome back!`, { duration: 4000 });
```

Comment:
why are you taking userName ? and not using that in toast? then why removing it from the localStorage?

Reply:
![image6](https://github.com/user-attachments/assets/7097fc42-af91-42f7-a6dd-b74226c7793e)
toast message correctly added user name correctly my wrong in login screen set item wrongly applied now i corrected that

Reply:
Can be modified to display as : Login successful. Welcome back, Akshaya !